packages/core/webpack.config.js

```diff
@@ -57,6 +57,11 @@ module.exports = {
           to: './',
           context: path.resolve(__dirname, '../', 'web-components'),
         },
+        {
+          from: 'components/**/*',
+          to: './',
+          context: path.resolve(__dirname, '../', 'web-components/dist'),
```

Comment:
Copying over the `dist-custom-elements` build allows it to be imported from the `component-library` package rather than just the `web-components` package.